orchestrator-salt/src/main/resources/salt-common/salt/fluent/scripts/cdp_logging_agent_check.sh

```diff
@@ -2,7 +2,7 @@
 
 : ${LOGFILE_FOLDER:="/var/log/cdp-logging-agent-doctor"}
 LIVENESS_THRESHOLD_SECONDS=86400 # 1 day in seconds
-BUFFER_LIMIT_BYTES=10485760 # 10 MB in bytes
+BUFFER_LIMIT_BYTES=1048576 # 1 MB in bytes
```

Comment:
Can we make an estimate that it will detect the errors after 4 hours of not sending HBs.

Reply:
it should be around 100K, note that the check running **daily** so it makes more sense to set it to ~500K, also note the chunk limit for fluentd is 600K (so it sends data when that is reached...although the timeout for that is 1 minute, so it should not reach 600K only if shipping is hanging ... could be decreased that as well, at least we won't send large chunks to dbus api, let's say we can decrease that to 100K, and decrease the buffer limit as well to 500K, but the chunk limit would mean we will hit the api 6x more times but with less data so not sure that helps if on connection issues)

Reply:
of course cron job can be modified as well to not run checks only at 01:00, but in every 4 hour, then using 100K for limit can make more sense